blacklight-cornell/jenkins-opts.sh

```diff
@@ -5,14 +5,17 @@ echo 'Use this script to run cucumber with the same options as jenkins.'
 echo 'But you also need to make sure that your .env file matches.'
 echo 'To run a specific suite, add file argument like features/catalog_search/results_list.feature'
 echo 'To run a specific test, add line argument like features/catalog_search/results_list.feature:317'
-OPT1="-t ~@oclc_request  -t ~@search_with_view_all_webs_match_box_with_percent "
+
+# Old set of options, looks like contains some duplicates
```

Comment:
Can we get rid of lines 8-26? We'll still have these old/commented out opts in version history if we ever need them.